roles/openshift_logging_kibana/templates/kibana.j2

```diff
@@ -47,6 +47,13 @@ spec:
 {% if kibana_memory_limit is not none and kibana_memory_limit != "" %}
               memory: "{{ kibana_memory_limit }}"
 {% endif %}
+            requests:
+{% if kibana_cpu_request is not none and kibana_cpu_request != "" %}
+              cpu: "{{ kibana_cpu_request }}"
+{% endif %}
+{% if kibana_memory_request is not none and kibana_memory_request != "" %}
+              memory: "{{ kibana_memory_request }}"
+{% endif %}
```

Comment:
Is it possible to have `requests` with no values in it?  Will this cause problems?